src/cljs/rems/status_modal.cljs

```diff
@@ -14,7 +14,7 @@
             [rems.text :refer [text]])
   (:require-macros [rems.guide-macros :refer [component-info example]]))
 
-(defn- format-errors [errors]
+(defn format-errors [errors]
```

Comment:
minor: This function is not used outside the namespace, so could keep it private.